src/elements/styles.js

```diff
@@ -358,6 +358,69 @@ class ApplyStyles {
       this.setStyle('placeholderFocus', 'display', 'none');
     }
   }
+
+  applyBackgroundImageStyles(target) {
+    if (
+      !this.styles?.['background_image_url'] &&
+      !this.mobileStyles?.['background_image_url']
+    ) {
+      return;
+    }
+
+    // Apply background image
+    this.apply(target, ['background_image_url'], (imageUrl) => {
+      return {
+        backgroundImage: `url(${imageUrl})`
+      };
+    });
+
+    // Apply background image display (contain, fit, etc)
+    const displayTargets = [
+      'background_image_display',
+      'background_image_size',
+      'background_image_size_x',
+      'background_image_size_y'
+    ];
+
+    this.apply(target, displayTargets, (...styles) => {
+      const [imageDisplay, imageSize, imageSizeX, imageSizeY] = styles;
```

Comment:
why not have these be explicit function parameters directly instead of aggregating then unaggregating them

Reply:
To prevent the `this.apply` from following on multiple lines due to line width constraints. Just cleans it up and makes it easier to read, IMO